libs/ui-toolkit/src/components/popover/popover.tsx

```diff
@@ -23,7 +23,7 @@ export const Popover = ({
         <PopoverPrimitive.Content
           data-testid="popover-content"
           align="start"
-          className="p-4 rounded bg-neutral-50 dark:bg-neutral-800 dark:text-neutral-200 border border-neutral-400"
+          className="p-4 rounded bg-neutral-100 dark:bg-neutral-800 dark:text-neutral-200 border border-neutral-400"
```

Comment:
Think these looked better lighter.

Reply:
The difference is minimal and would rather not introduce an 11th grayscale color right now in the interest of maintenance